src/models/userModel.ts

```diff
@@ -0,0 +1,49 @@
+import { DataTypes, Model, UUIDV4 } from "sequelize";
+import { sequelize } from "../db/db";
+
+interface UserAttributes {
+	id: string;
+	firstName: string;
+	lastName: string;
+	email: string;
+	password: string;
+}
+
+interface UserModel extends Model<UserAttributes>, UserAttributes {}
+
+const User = sequelize.define<UserModel>(
+	"User",
+	{
+		id: {
+			type: DataTypes.UUID,
```

Comment:
Please leave the id as the sequential id provided by the DB. You can create a different field for the uuid.

Reply:
Done